esp-wifi/src/wifi/mod.rs

```diff
@@ -2563,6 +2759,20 @@ impl<'d> WifiController<'d> {
         }
     }
 
+    /// Set CSI configuration and register the receiving callback.
+    #[cfg(csi_enable)]
+    pub fn set_csi(
+        &mut self,
+        mut csi: CsiConfig,
+        cb: alloc::boxed::Box<dyn FnMut(crate::wifi::wifi_csi_info_t)>,
```

Comment:
Oh actually, does this need to be `Send`?

Like this
```suggestion
        cb: alloc::boxed::Box<dyn FnMut(crate::wifi::wifi_csi_info_t) + Send>,
```